decidim-admin/app/views/decidim/admin/organization_appearance/form/_colors.html.erb

```diff
@@ -1,6 +1,16 @@
 <div class="card">
   <div class="card-divider">
-    <h2 class="card-title"><%= t ".colors_title" %></h2>
+    <h2 class="card-title"><%= t ".colors_title" %> </h2>
+  </div>
+  <div class="callout warning" data-closable>
+    <svg role="img" class="icon--warning icon">
+      <title></title>
+    </svg>
+    <p><%= t ("from.colors.warning_colors") %> </p>
```

Comment:
This is a good case for using two i18n features: [passing variables](https://guides.rubyonrails.org/i18n.html#passing-variables-to-translations) and [using HTML](https://guides.rubyonrails.org/i18n.html#using-safe-html-translations).